src/scry/completion/require_module_context.cr

```diff
@@ -0,0 +1,37 @@
+module Scry::Completion
+  class RequireModuleContext < Context
+    RELATIVE_IMPORT_REGEX = /^\./
+
+    def initialize(@import : String, @text_document : TextDocument)
+    end
+
+    def find
+      case @import
+      when RELATIVE_IMPORT_REGEX
```

Comment:
`when .starts_with? '.'` would be faster here I think (and less regex is almost always better :smile: )